src/actions/channels.js

```diff
@@ -568,6 +561,8 @@ export function viewChannel(channelId) {
             return null;
         }
 
+        setServerVersion(Client4.getServerVersion())(dispatch, getState);
```

Comment:
Does it update the store anytime it changes in the client? I wasn't seeing that happen so I just had it get set on a regular call like view channel